dts/common/yaml/uart.yaml

```diff
@@ -22,4 +22,9 @@ properties:
       category: required
       description: Clock gate information
       generation: define
+  - label:
+      type: string
+      category: optional
+      description: Human readable string describing the device
```

Comment:
Is this description correct? I'd say in usage here it's "API identifier of the device".